backend/src/main/java/com/encora/backend/model/Task.java

```diff
@@ -0,0 +1,79 @@
+package com.encora.backend.model;
+
+import java.util.Date;
+
+public class Task {
+    private Long id;
+    private boolean state;
+    private Date doneDate;
+    private Date creationDate;
+    private Date dueDate;
+    private String priority;
```

Comment:
Are the possible values for this field fixed, or can it contain any value? If they are fixed, have you considered using an enum?